### PR TITLE
Update Chinese

### PR DESCRIPTION
**Pull request by Orange23333**
# To Aministrator

_Because I can't connect to Discord, so I leave some messages here._

 **It is suggested to change the file name to `zh-CN.ts`, and change the name of the dictionary form `ZH` to `ZH_CN`.** 

 **I use the converter to make a simple conversion from `simplified Chinese` to `traditional Chinese`. This will help people who are used to using traditional Chinese to read texts. Further localization may be needed.** 

# 修改(Change)

1. `Fashion`: `塑造`(characterization) -> `时装`(fashionable dress)
2. `LumberMill`: `伐木工坊`(lumbering workshop) -> `锯木厂`(sawmill)
3. `PowerBank`: `电力银行(Power bank)` -> `电池组(Battery group)`
4. 个人建议(Personal suggestion): `发电`+站/厂(as a place) `矿`+场(as a place)
5. `生产(Production)``数量(Amount)`等用词
6. `建筑(Building)``建造(Build)`等用词
7. `...怎样的` -> `怎样的...` (I think `...` may mean `buildings`, and the `nounce` should be behind `adjective` in Chinese)
8. 移除部分`空格(white space)`
9. 根据实际游玩，意译部分内容(According to the actual play, free translation part of the content)
10. 对于更新部分的翻译(For the translation of the update)
11. _略(.etc)_

----------

# 留言(Crap)

_撞车了（虽然按照我的速度来说是必然的事），今天更新发现已经被翻译好了，也好我估计我得翻译个把个星期才能出来。那我就稍微提出一点点意见来帮帮忙好了(=▽=) (_Some pleasantries to other translators XD_)_

_虽然觉得`zh-TW`这种说法怪怪的（`zh-SP`和`zh-TD`更好一些），毕竟有`单纯的繁体`、`台式正繁`、`港澳繁体`，但这是难改变的标准。(Although I think that the term `zh-TW` is strange (`zh-SP` and `zh TD` are better), after all, there are `pure traditional Chinese`, `desktop Zhengfan`, `hong Kong and Macao traditional Chinese`, but this is a standard that is difficult to change.)_

### CODE REVIEW

**Comment by Orange23333** [comment]
@LazySu @snewpeap

**Comment by Orange23333** [comment]
@LazySu @snewpeap 请检查谢谢！

**Comment by Orange23333** [comment]
@LazySu @snewpeap 看看

File: zh_CN.ts [hunk 1]
```diff
@@ -276,8 +276,8 @@ export const ZH = {
     OfflineEarningTimeDesc: "您能够获取离线收益的时长",
     BuyPermit: "购买许可证",
     NotEnoughBuildingPermit: "没有足够的建筑许可证，请购买更多来建造新的建筑物",
-    MovingAverage: "平均移动",
-    MovingAverageDesc: "显示资源变化的移动平均值，而不是实时变化",
+    MovingAverage: "平均变动",
+    MovingAverageDesc: "显示资源变化的平均值，而不是实时变化量",
```
Comment (snewpeap) [inline zh_CN.ts:280]: moving average好像是个专有名词（[百度百科](https://baike.baidu.com/item/滑动平均/2640979)|[Wikipedia](https://zh.wikipedia.org/wiki/%E7%A7%BB%E5%8B%95%E5%B9%B3%E5%9D%87)），考虑到原文是"Show moving average of change"，建议如下：
Moving average seems to be a term. Considering that the original text is "Show moving average of change", the suggestions are as follows:
```suggestion
    MovingAverage: "移动平均",
    MovingAverageDesc: "显示资源变化的移动平均值，而不是实时变化量",
```

File: zh_CN.ts [hunk 2]
```diff
@@ -297,11 +297,11 @@ export const ZH = {
...
-    HighlightStockpileModeOn: "...开启库存模式的",
-    HighlightWarehouseOnly: "...只供应仓库的",
-    HighlightProductionCycleNotDefault: "...生产周期 > 1的",
-    HighlightMaxInputDistanceNotDefault: "...最大输入距离 < ∞的",
+    HighlightTurnedOff: "被关闭的...",
+    HighlightStockpileModeOn: "开启库存模式的...",
+    HighlightWarehouseOnly: "只供应仓库的...",
+    HighlightProductionCycleNotDefault: "生产周期 > 1的...",
+    HighlightMaxInputDistanceNotDefault: "最大输入距离 < ∞的...",
     HighlightNotMakingProfit: "...不盈利的",
```
Comment (snewpeap) [inline zh_CN.ts:305]: 你忘了这个
You forgot this
```suggestion
    HighlightNotMakingProfit: "不盈利的...",
```

File: zh_CN.ts [hunk 3]
```diff
@@ -648,28 +648,28 @@ export const ZH = {
...
-    CoalMine2xOutput: "Industrial Revolution",
-    CoalMine2xOutputDesc: "Coal mines produce 2x coal but use 50% more power and coal transport costs 50% more fuel",
-    AlMine2xOutput: "Aluminum Smelting",
-    AlMine2xOutputDesc: "Aluminum mines produce 2x aluminum and use 2x more power",
+        "硅矿场生产2倍的硅，但用电量增加50%，运输硅的燃料消耗增加50%",
+    CoalMine2xOutput: "工业革命",
+    CoalMine2xOutputDesc: "煤矿场生产2倍的煤，但用电量增加50%，运输煤的燃料消耗增加50%",
+    AlMine2xOutput: "铝电解",
+    AlMine2xOutputDesc: "铝矿场产生2倍的铝，但消耗2倍的电力",
     LoggingCamp2xOutput: "Deforestation",
```
Comment (snewpeap) [inline zh_CN.ts:669]: 你忘了这个
You forgot this
```suggestion
    LoggingCamp2xOutput: "毁林",
```

File: zh_CN.ts [hunk 4]
```diff
@@ -648,28 +648,28 @@ export const ZH = {
...
-    LoggingCamp2xOutputDesc: "Logging camps produce 2x wood and wood transport costs 2x fuel",
-    HideResourceMovement: "Hide Resource Movement",
+    LoggingCamp2xOutputDesc: "伐木场产生2倍的铁，但运输木头消耗2倍的燃料",
```
Comment (snewpeap) [inline zh_CN.ts:670]: 应该是“2倍的木头”不是“铁”
Should be "2x wood" not "iron"
```suggestion
    LoggingCamp2xOutputDesc: "伐木场产生2倍的木头，但运输木头消耗2倍的燃料",
```

**Review by snewpeap — CHANGES_REQUESTED** [review]
社区感谢你。在下有一些建议，后附。
The community thank you. There are some suggestions below, attached.

**Comment by snewpeap** [comment]
还有一些重要但尚未保持一致的名词翻译，列举如下：
There are still some important but not yet consistently translated nouns, as listed below:
- tile -> 瓷砖，地砖，格；
- policy -> 策略，政策；
- capacity -> 产能，能力，容量；
- etc.
我不知道统一到哪个翻译更好，大伙可以给给建议。
I don't know which translation to unify to. Need suggestions.

**Comment by insraq** [comment]
@snewpeap @Orange23333 Wow, thanks for your contribution and review! I have merged all the changes and feel free to make further corrections.

**Comment by Orange23333** [comment]
> 社区感谢你。在下有一些建议，后附。
> The community thank you. There are some suggestions below, attached.
谢谢！

**Comment by Orange23333** [comment]
谢谢指正与建议！

**Pull request merged**